decidim-assemblies/app/cells/decidim/assemblies/assembly_g_cell.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Assemblies
+    # This cell renders the Medium (:m) assembyl card
+    # for an given instance of an Assembly
+    class AssemblyGCell < Decidim::CardMCell
```

Comment:
This seems to be a copy of AssemblyMCell.

Reply:
Yes, the idea is to keep the old cells (:s and :m) and the new (:l and :g) in parallel and replace completely the old ones with the new when redesign finished. In this PR the card m has been copied and the the show has been changed but not fully integrated. The correct version of this cell is integrated in the [card-g PR](https://github.com/decidim/decidim/blob/0c90b0a66dd9b61651f2fb38390f6c1d293717f4/decidim-assemblies/app/cells/decidim/assemblies/assembly_g_cell.rb) where the specific card-g of assemblies inherits from `Decidim::CardGCell` and the show.erb file is not necessary

In everything related to card-g the corresponding PR will substitute the code shown in this PR